app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable that tracks all instances of `Animal`. Ensure that instances are properly removed from this list when they are no longer needed to avoid memory leaks.